src/IBM.Watson.Discovery.v2/DiscoveryService.cs

```diff
@@ -1667,7 +1825,7 @@ public DetailedResponse<AnalyzedDocument> AnalyzeDocument(string projectId, stri
 
                 if (file != null)
                 {
-                    var fileContent = new StreamContent(file);
+                    var fileContent = new ByteArrayContent(file.ToArray());
```

Comment:
I'm taking your word that this is a compatible non-breaking change

Reply:
It _seems_ like it's an internal implementation detail to creating the file, but just want to be sure.

Reply:
good catch, I fixed it